source/getting-started/automation.markdown

```diff
@@ -3,43 +3,31 @@ title: "Automating Home Assistant"
 description: "A quick intro on getting your first automation going."
 ---
 
-Once your devices are set up, it's time to put the cherry on the pie: automation. In this guide we're going to create a simple automation rule to turn on the lights when the sun sets. Of course, this assumes that you have set up an integration that provides a light at this point.
+Once your {% term devices %} are set up, it's time to put the cherry on the pie: {% term automation %}. In this guide we're going to create a simple automation rule to turn on the lights when the sun sets. Of course, this assumes that you have set up an integration that provides a light at this point.
 
-In the user interface, click **Settings** in the sidebar, then click **Automations & Scenes**. You will now see the automation screen from which you can manage all the automations in Home Assistant.
+In the user interface in the side bar, click **{% my automations title="Settings -> Automations & Scenes" %}**. You will now see the automation screen from which you can manage all the automations in Home Assistant.
```

Comment:
- The sidebar is a single thing (not the bar on the side, but the sidebar).
- Replace `->` by `>`, which we are trying to make a standard.

```suggestion
In the user interface in the sidebar, click **{% my automations title="Settings > Automations & Scenes" %}**. You will now see the automation screen from which you can manage all the automations in Home Assistant.
```

Reply:
Got it. In terms of the changes like -> to >, is there a place where I can follow these discussions or latest conventions?

Reply:
There seem to be different styles in use: >>, -> and >.
we are trying to follow the Microsoft Style Guide, which uses >.
https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui